src/components/CreateEvent/CreateEvent.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { CreateEventWrapper, Form } from './styled';
+import { parseISO, formatISO, format } from 'date-fns';
+
+
+function CreateEvent() {
```

Comment:
We should use a `Page` suffix for all page components. This is to distinguish them from a type or a command. In this case, you might think that this function is creating an event, not that it is a page for creating events.